Fix broken links, typos, and other small errors (#796)
timescale/docs: timescaledb/how-to-guides/connecting/about-connecting.md

    it is marked as `Running`.
1.  Click the name of the service to see its connection information. Copy the
    `Service URL`, which you will need when connecting.
1.  If you don't know the password for the service, navigate to the `Operations`
    tab, and click `Reset password`. You can choose your own password or allow
    Timescale Cloud to generate a secure password for you. Keep a copy of your
    new password, which you will need when connecting.

<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-connection-info.png" alt="View Timescale Cloud connection info"/>

</procedure>

## Find connection details in Managed Service for TimescaleDB
To retrieve your connection details from a running Managed Service for
TimescaleDB service:

<procedure>

### Finding connection details in Managed Service for TimescaleDB
1.  Sign in to the [Managed Service for TimescaleDB portal][mst-portal].
1.  In the `Services` tab, find the service you want to connect to, and check
    it is marked as `Running`.
1.  Click the name of the service to see its connection information. Copy the

>     `host`, `port`, and `password`. You will need these when connecting.

Reviewer: ```suggestion
    `host`, `port`, and `password`. You need these to connect.
```
